src/Xamarin.Android.Build.Tasks/Tasks/ConvertCustomView.cs

```diff
@@ -75,7 +78,20 @@ public override bool Execute ()
 					}
 				}
 			}
-
+			var output = new List<ITaskItem> ();
+			foreach (var file in processed) {
+				ITaskItem resdir = ResourceDirectories?.FirstOrDefault (x => file.StartsWith (x.ItemSpec)) ?? null;
+				var hash = resdir?.GetMetadata ("Hash") ?? null;
+				var stamp = resdir?.GetMetadata ("StampFile") ?? null;
```

Comment:
Since we are doing `resdir?` multiple times, would a `if (resdir == null) continue;` be better?

Does it also need to log something in that case?